docs/3-managed-service.md

```diff
@@ -4,158 +4,106 @@ The hosted version of Hush Line is the first free and open-source anonymous tip-
 
-## Register
+## Register & Login
 
-Before public launch of Hush Line, you'll need an invite code to create an account. Codes are distributed upon request. Once you have the invite code, register your account at https://tips.hushline.app/register.
+First, register and log in to your account at https://tips.hushline.app/register.
 
-<img src="img/auth.register.png">
+To protect your privacy, we don't require any PII, including your email address or phone number.
 
-## Log In
-
-After creating your account, you'll be redirected to the login page. Enter the username and password you just created.
-
-Note: To protect your privacy, we don't require any PII, including your email address or phone number.
-
-<img src="img/auth.login.png">
+![Register](https://github.com/user-attachments/assets/f3846273-e475-42c2-aa86-806a54e32e72)
 
 ## Verified Accounts
 
-At Hush Line, we're committed to building a trusted community where communication is secure and identities are verified. Our verified accounts feature is designed to ensure that messages reach their intended recipients, fostering a reliable environment for all users. Verified accounts are specially available for:
+Our verified accounts feature is designed to ensure that messages reach their intended recipients. Verified accounts are specially available for:
 
 - 🕵️ **Journalists** - Reporters, correspondents, and investigative journalists.
 - 📰 **Newsrooms** - Official accounts for newspapers, TV stations, or online news portals.
 - ✊ **Activists** - Individuals or groups advocating for social, environmental, or political causes.
 - 📸 **Public Figures** - Politicians or other noteworthy public individuals.
 - 📊 **Businesses** - Companies, small businesses, or startups aiming to communicate with their audience.
 
-<img src="img/verified-account.png">
+![Profile](https://github.com/user-attachments/assets/b34bb0c8-ad9e-4aef-a412-5085c61bb230)
 
 ### Requesting Verification
 
 To ensure your account is recognized as authentic, users belonging to the categories listed above can apply for verification. Follow these steps to initiate the verification process:
 
-1. **Send a Message:** Use the contact form in the app to reach out to us. Include your name or organization's name and a preferred method of contact (email or phone number). We'll get back to you to schedule a verification meeting.
-2. **Prepare Your Documents:** To verify your identity or your authority to represent an organization, please include the relevant documents for your role:
+1. **First, add a verified URL to your Hush Line profile:** In Hush Line settings you can add up to four additional fields - a URL, Signal username, phone number, or anything else. To verify a URL, add it to your Hush Line extra fields using `https://` - `https://scidsg.org`, for example. Then, add a link to your Hush Line address on the page for that URL with `rel="me"` in the markup. For example:
```

Comment:
Does this mean that we're eliminating the manual verification process? If we're just relying on verified URLs, what's to stop someone from setting up a ring of fake profiles that are all linked to "verify" each other?